src/app/core/header/navbar/login-dialog/login-form/login-form.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { MatDialogRef } from '@angular/material';
+
+import { AuthService } from 'src/app/services/auth.service';
+import { LoginDialogComponent } from '../login-dialog.component';
+import { LoginDialogService } from './../login-dialog.service';
+
+@Component({
+  selector: 'dialog-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.scss']
+})
+
+export class LoginFormComponent {
+  public hide: boolean = true;
+  public email: FormControl = new FormControl('', [Validators.required, Validators.email]);
+  public password: string;
+  public loginForm: FormGroup;
+
+  constructor(public dialogRef: MatDialogRef<LoginDialogComponent>,
```

Comment:
bad formatting.
Services should be aligned.

`  constructor(
    private service1: Service1,
    private service2: service2,
  ) { }`